src/ADAL.PCL.Android/CryptographyHelper.cs

```diff
@@ -34,6 +34,11 @@ internal class CryptographyHelper : ICryptographyHelper
     {
         public string CreateSha256Hash(string input)
         {
+            if (string.IsNullOrWhiteSpace(input))
+            {
+                return null;
+            }
+
             return string.Empty;    // TODO: Update
```

Comment:
As discussed offline, maybe also ask @kpanwar as to why the Android `CryptographyHelper` does not have an implementation (whereas iOS does – i.e., the functionality is supported in Mono-based runtimes).
Probably should be a work item to cover this `//TODO`